scripts/DCOSWindowsAgentSetup.ps1

```diff
@@ -238,36 +238,14 @@ function New-DockerNATNetwork {
     Write-Output "Created customnat network with flag: com.docker.network.windowsshim.disable_gatewaydns=true"
 }
 
-function Generate-MesosHttpHealthCheckImage{
-    $AgentOSBuild = [System.Environment]::OSVersion.Version
-
-    if ($AgentOSBuild.build -eq $WINDOWS_SERVER_RS4_BUILD_NUMBER) {
-        $imageTag=$WINDOWS_SERVER_RS4_DOCKER_IMAGE_TAG
-    } elseif ($AgentOSBuild.build -eq $WINDOWS_SERVER_RS3_BUILD_NUMBER) {
-        $imageTag=$WINDOWS_SERVER_RS3_DOCKER_IMAGE_TAG
-    } else {
-        $imageTag="latest"
-    }
-    # generate dockerfile from template
-    Write-output "Agent node Windows build number: $imageTag"
-    $context = @{
-        "imageTag" = ("$imageTag")
-    }
-    $TEMPLATES_DIR = Join-Path "$SCRIPTS_DIR\scripts" "templates" 
-    $dockerfileTemplate = "$TEMPLATES_DIR\mesos-http-health\dockerfile.template"
```

Comment:
If we remove this, do we need anymore the template from: https://github.com/dcos/dcos-windows/tree/master/scripts/templates/mesos-http-health ?